### PR TITLE
temp fix for issue 259

### PR DESCRIPTION
**Pull request by jasonbrooks**
temp fix for issue #259 

@cgwalters @kbsingh

### CODE REVIEW

**Comment by cgwalters** [comment]
Seems likely to work.  It's not going to hurt at least, if it doesn't we can try again.

**Comment by jasonbrooks** [comment]
I'll see if it breaks on ci

**Comment by jasonbrooks** [comment]
OK, it didn't break there.

**Comment by miabbott** [comment]
This doesn''t appear to fix the problem:

```
# atomic host status
State: idle
Deployments:
● centos-atomic-host:centos-atomic-host/7/x86_64/standard
       Version: 7.20170209 (2017-02-10 00:54:47)
        Commit: d433342b09673c9c4d75ff6eef50a447e73a7541491e5197e1dde14147b164b8
        OSName: centos-atomic-host
  GPGSignature: 1 signature
                Signature made Fri 10 Feb 2017 01:06:18 AM UTC using RSA key ID F17E745691BA8335
                Good signature from "CentOS Atomic SIG <security@centos.org>"

  centos-atomic-host:centos-atomic-host/7/x86_64/standard
       Version: 7.20161006 (2016-10-05 23:28:52)
        Commit: 2ad34dba5286d69de71b61678d5190b246e7836beada3cdfa18f7bfa4ae3c3f0
        OSName: centos-atomic-host
  GPGSignature: 1 signature
                Signature made Wed 05 Oct 2016 11:39:15 PM UTC using RSA key ID F17E745691BA8335
                Good signature from "CentOS Atomic SIG <security@centos.org>"
-bash-4.2# atomic host upgrade
Updating from: centos-atomic-host:centos-atomic-host/7/x86_64/standard
error: No detached metadata found for GPG verification
```

**Comment by jasonbrooks** [comment]
@kbsingh needs to build the new tree and put in place before we know if it worked

**Comment by dmartinpro** [comment]
Still got the same error while trying to check for upgrade from centos-atomic-host 7.20170309

**Comment by jasonbrooks** [comment]
@dmartinpro We're releasing a new tree today, you should be able to upgrade now

**Comment by michaelschefczyk** [comment]
Can someone please be so kind to state in plain English how to enable a cluster of centos-atomic-host 7.20170309 to upgrade? It always provides error: No detached metadata found for GPG verification
Entering, atomic host status yields Good signature from "CentOS Atomic SIG <security@centos.org>"
Modifying /ostree/repo/config by adding gpg-verify=false does not change this.

**Comment by jasonbrooks** [comment]
@michaelschefczyk Can you try running `atomic host deploy 7.20170405`

**Comment by michaelschefczyk** [comment]
Yes, that does work - thank you very much for poiting this out!!
